br/pkg/task/backup_ebs.go

```diff
@@ -219,7 +186,7 @@ func RunBackupEBS(c context.Context, g glue.Glue, cmdName string, cfg *BackupEBS
 	// Step.2 starts call ebs snapshot api to back up volume data.
 	// NOTE: we should start snapshot in specify order.
 
-	progress := g.StartProgress(ctx, cmdName, int64(storeCount), !cfg.LogProgress)
+	progress := g.StartProgress(ctx, "ebs backup", int64(storeCount), !cfg.LogProgress)
```

Comment:
what about snapshot backup? since we will also move to other platform e.g gcs.

Reply:
change to `backup`